### DISCUSSION TITLE
Better readability of add-ons states in Supervisor

### DISCUSSION THREAD

**Discussion by schmurtzm** [home-assistant/frontend#11263] [Other feature requests]
## The request
Better readability of add-ons states in Supervisor. At that time it is hard to have an effective quick view because the cases box are too similar. The thin colored line at the top of the box is not enought.

**Idea :** 
Color the full box for better readability.
As a simple color is not explicit you could add an icon associate with each color.

**A (very) quick and dirty  example :** 
![supervisor home assistant proposition](https://user-images.githubusercontent.com/7110113/82755284-e93cf280-9dd2-11ea-8fae-ded14d15c6ec.jpg)

But it still missing a lot of information, it's not compact and you can't manage anything befor clicking on an add-on.

## The alternatives

**This is another proposition :** 

![supervisor home assistant](https://user-images.githubusercontent.com/7110113/82755288-ea6e1f80-9dd2-11ea-9b71-fa79a188c2bb.jpg)

## Additional information

Some details explained : 

![supervisor home assistant with comments](https://user-images.githubusercontent.com/7110113/82755287-ea6e1f80-9dd2-11ea-8257-44825cff3098.jpg)

With this view you can unify the store and the dashboard and you're able to make a lot of things on one screen.
You could also add another coloum with start on boot state.
@SeanPM5

**Comment by zsarnett** [comment]
The full color card doesn't look good at all. We should maybe just make the color border larger.

**Comment by bramkragten** [comment]
I think the icon could work

**Comment by SeanPM5** [comment]
How about filtering "installed add-ons" out of the store entirely?

Then the user doesn't have to figure out what the blue color or icon means. And it will be faster to actually find things in the store because it's not cluttered with stuff you already own. And it would give the tabs a clear separation - dashboard is for stuff you have, store is for finding/installing new things. 

I don't like the full color cards, but the icon idea for update available and stopped seems OK to me. There is a potential for multiple statuses though - you can have an add-on that is stopped _and_ has an update available. So that would need to be figured out, either display two icons or pick one that gets priority.

**Comment by ludeeus** [comment]
How about no cards?
![image](https://user-images.githubusercontent.com/15093472/94177617-20630380-fe9a-11ea-9906-73635dbc07b4.png)
![image](https://user-images.githubusercontent.com/15093472/94177702-4688a380-fe9a-11ea-9042-e0f21f5b9fcd.png)

**Comment by bramkragten** [comment]
That looks nice, how would it work on really wide screens?

**Comment by ludeeus** [comment]
> how would it work on really wide screens?

Since it's not built, however we want it to look :wink: 
For now, while testing, it's like this (2560x1440):
![image](https://user-images.githubusercontent.com/15093472/94179926-8735ec00-fe9d-11ea-8e51-3251a302c7f1.png)

**Comment by schmurtzm** [comment]
> 
> How about no cards?

Why not ! :)
It create a lot of space so it could be nice to have more columns with control here : 

1. -  A button to make a direct update
2. - Display "current version number" --> "updarted version number" .    For example "**update avaible : 5.1.0 --> 5.2.0**" ,click on it to update with a confirmation
3. -  A button stop/play and a button restart (to avoird to have to open the card to make these steps)
4. -  May be color the full row when an update is available ? 
5. -  When running an update be able to see the state on this screen without refresh ("Running" -> updating" -> "Running")

**Comment by mrand** [comment]
If this hasn't already been resolved along with the menu restructuring (I kinda thing it has),  I really hope a grouping method gains favor so that this can be done without relying on colors.  

Ignoring color blindness (or low-sight) usability related issues that should be considered, it will takes away part of the concept of having a theme in the first place when things start getting locked down - especially because there's a chance those locked down things may actually conflict with the theme.  i.e. what happens when a widely used color of the theme matches one of the red or yellow or green "traffic light" colors?  Now because it's locked down, there's no way to make it actually stand out.

**Comment by ncodee** [comment]
+1 with @ludeeus implementation.
